3-extra/1-guess-the-number-project/script.js

```diff
@@ -1,16 +1,31 @@
-let randomNumber = Math.floor(Math.random() * 100 + 1);
+let randomNumber,
+  tries = 0;
 
+let output = document.querySelector(".final-output");
+
+const generateRandomNumber = () => {
+  randomNumber = Math.floor(Math.random() * 100 + 1);
+};
+
+console.log(randomNumber);
+generateRandomNumber();
 function guessNumber() {
   //Collect input from the user
   let guess = document.querySelector(".inputs-Values").value;
 
   //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
-
-  //If the users guess is higher than the random number print Number is too high, try again (hint use .final-out class to print)
-
-  //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
-
-  //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
+  guess < 1 || guess.length < 1 || guess > 100
+    ? (document.querySelector(".final-output").innerHTML =
+        "Please enter a number between 1 and 100")
+    : //If the users guess is higher than the random number print Number is too high, try again (hint use .final-out class to print)
+    guess > randomNumber
+    ? (output.innerText = `Number is too high, try again`)
+    : //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
+    guess < randomNumber
+    ? (output.innerText = `Number is too low, try again`)
+    : //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
+      (output.innerText = `Guess is correct. You win!`);
+  document.querySelector(".Tries-output").innerText = `${++tries} attempt`;
 }
```

Comment:
Hey Omid, I like your use of ternary operations here! I was thinking of going back and doing the same with mine :)